### ISSUE TITLE
Fix aspect ratio

### ISSUE THREAD

**Issue by robertlugg** [agnivade/shimmer#5]
The aspect ratio of the image appears set by the browser window.  Instead, have the image left at a fixed aspect ratio.

**Comment by agnivade** [comment]
Yeah, currently the width is hardcoded to 49%, to prevent large images from not appearing side-by-side. Some more CSS magic sprinkle needed. PRs are most welcome.
